tools/tarantool-gdb.py

```diff
@@ -2354,3 +2481,67 @@ def invoke(self, arg, from_tty):
             TtListPrinter.reset_config()
 
 TtListSelect()
+
+
+class TtListWalk(gdb.Command):
+    """
+tt-list-walk
+Display list entry LIST_EXP refers to or the entire list if LIST_EXP refers
+to the list head.
```

Comment:
This command has an exactly the same description as `tt-list`. It's unclear to me, as a user, which one I should pick. Please elaborate on that in the command description: why we need two commands to walk over a list; which one is preferable in a particular scenario.